pkg/restapi/verifier/operation/operations.go

```diff
@@ -182,6 +186,35 @@ func (o *Operation) getProfileHandler(rw http.ResponseWriter, req *http.Request)
 	commhttp.WriteResponse(rw, profile)
 }
 
+// DeleteVerifierProfile swagger:route DELETE /verifier/profile/{id} verifier deleteProfileReq
+//
+// Deletes verifier profile.
+//
+// Responses:
+// 		default: genericError
+//			200: emptyRes
+func (o *Operation) deleteProfileHandler(rw http.ResponseWriter, req *http.Request) {
+	profileID := mux.Vars(req)["id"]
+
+	err := o.profileStore.DeleteProfile(profileID)
+
+	if err != nil {
+		if strings.Contains(err.Error(), storage.ErrValueNotFound.Error()) ||
+			strings.Contains(err.Error(), "Not Found: deleted") {
```

Comment:
Is this specific to couchdb ?

Reply:
yes.. should i change corestore instead to ignore "Not Found:deleted"?

Reply:
@rolsonquadras

Reply:
I would expect all store to be consistent with error messages ie - storage.ErrValueNotFound, rather than client handling individual store impl error messages. 

We can add this string check to couchdb and throw storage.ErrValueNotFound. This can be a separate PR - update edge-core and remove this check here.

Reply:
@grace0925 Also, when you rework the error handling per @rolsonquadras's comments, you should use `errors.Is(storage.ErrValueNotFound)`